db/database.py

```diff
@@ -0,0 +1,39 @@
+### This is the python file I use to work with the sqlite3 database, since sqlite3 is integrated with python
+
+import sqlite3
+
+conn = sqlite3.connect('items.db')
+c = conn.cursor()
+
+# Create items table
+c.execute("""CREATE TABLE items (
```

Comment:
Here maybe it's better to use a "CREATE TABLE IF NOT EXISTS" 😊？

Reply:
yesssss good idea ^^